WcaOnRails/spec/models/competition_wcif_spec.rb

```diff
@@ -0,0 +1,177 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+ONE_MINUTE_IN_CENTISECONDS = 60*100
```

Comment:
I feel weird naming a constant not entirely in caps, so I don't think I want to change the variable name, although I could see "CENTIS" instead of "CENTISECONDS".

Monkeypatching sounds cute! Would you do this as a monkeypatch of [ActiveSupport::Duration](http://api.rubyonrails.org/classes/ActiveSupport/Duration.html)?